src/repr/src/row.rs

```diff
@@ -2378,6 +2379,49 @@ impl Default for RowArena {
     }
 }
 
+/// A thread-local row, which can be borrowed and returned.
+/// # Example
+///
+/// Use this type instead of creating a new row:
+/// ```
+/// use mz_repr::SharedRow;
+///
+/// let binding = SharedRow::get();
+/// let mut row_builder = binding.borrow_mut();
+/// ```
+///
+/// # Panic
+///
+/// [`SharedRow::get`] panics when trying to obtain multiple references to the shared row.
```

Comment:
I don't think that's accurate for the current code? Did you have the `borrow_mut` call inside `get` previously?

Reply:
I had an option that I would unwrap, but now it's borrowing to clear the row, so it would panic when obtaining two references and borrowing both. I'll make that clear in the comment.